src/main/java/com/charan/parkingsystem/parking/ParkingSlot.java

```diff
@@ -0,0 +1,43 @@
+package com.charan.parkingsystem.parking;
+
+import com.charan.parkingsystem.Types;
+
+public class ParkingSlot {
+    private final int slotID;
+    private Types.VehicleType spotType;
+    private boolean isFree = true;
+
+    public ParkingSlot(int id) {
+        this.slotID = id;
+    }
+
+    public ParkingSlot(int id, Types.VehicleType type) {
+        this.slotID = id;
+        this.spotType = type;
+    }
+
+    public int getSlotID() {
+        return slotID;
+    }
+
+    public boolean getIsFree() {
+        return isFree;
+    }
+
+    public void setFree() {
+        isFree = true;
+    }
+
+    public void setOccupied() {
```

Comment:
setter methods should be receiving an input to set a variable. Rename method or pass argument.